apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
+    test('recipients.add() parenthesis ', function(done) {
+        recipients.on('add', function(count, added) {
```

Comment:
I am not sure if what I have written is write. Any chance you could point out anything I am doing wrong? Or explanation on this particular case?